python/test/integration/test_mean.py

```diff
@@ -0,0 +1,32 @@
+import unittest
```

Comment:
We're already using pytest in other parts of the library, is there any reason not to use pytest across the board?
You can mark integration tests separately:
https://docs.pytest.org/en/latest/example/markers.html

Reply:
Thanks, I'll change this, I always defaulted to unittest and when it didn't appear I installed it without thinking about other options.